lib/elixir/lib/calendar/time.ex

```diff
@@ -558,6 +558,44 @@ defmodule Time do
     Calendar.ISO.iso_days_to_unit(iso_days, :microsecond)
   end
 
+  @doc """
+  Shifts a Time by given Duration according to its calendar.
+
+  Check `Calendar.ISO.shift_time/4` for more information.
```

Comment:
This is the primary API, all of the docs should be here. We shouldn't point most users to `Calendar.ISO` (rather only the ones who need custom calendars).